pkg/apis/pipeline/v1alpha1/run_types.go

```diff
@@ -201,7 +201,7 @@ func (r *Run) GetStatusCondition() apis.ConditionAccessor {
 
 // GetGroupVersionKind implements kmeta.OwnerRefable.
 func (*Run) GetGroupVersionKind() schema.GroupVersionKind {
-	return SchemeGroupVersion.WithKind(pipeline.RunControllerName)
+	return SchemeGroupVersion.WithKind(pipeline.CustomRunControllerName)
```

Comment:
In issue #6528 I suggested that `RunControllerName ` needs to be removed, but I didn't realise that is still used to `run_types.go` - so I think this one should stay as `RunControllerName`.
Everything else is ok since `Runs` are not supported by the `PipelineRun` controller anymore.